ee/clickhouse/views/person.py

```diff
@@ -17,31 +17,10 @@ def funnel_correlation(self, request: request.Request, **kwargs) -> response.Res
         if request.user.is_anonymous or not self.team:
             return response.Response(data=[])
 
-        results_package = self.calculate_funnel_correlation_persons(request)
-
-        if not results_package:
-            return response.Response(data=[])
-
-        people, next_url, initial_url, missing_persons = results_package["result"]
-
-        return response.Response(
-            data={
-                "results": [{"people": people, "count": len(people)}],
-                "next": next_url,
-                "initial": initial_url,
-                "is_cached": results_package.get("is_cached"),
-                "last_refresh": results_package.get("last_refresh"),
-                "missing_persons": missing_persons,
-            }
-        )
+        return _respond_with_cached_results(self.calculate_funnel_correlation_persons(request))
```

Comment:
Changing how the cached result is used means that all existing insights will start failing for upto 7 days 😬 . (oh, the troubles with cache invalidation).

You could clear the cache, or keep the cached result the same, and add the mods to `_respond_with_cached_results`, vs to `calculate_funnel_*`

Reply:
Yeah I just noticed this as well. Annoying. Fix coming up